src/components/value-list/interfaces.ts

```diff
@@ -0,0 +1 @@
+export type DragStatus = "start" | "activated" | "newPosition" | "currentPosition";
```

Comment:
Suggestion: `idle | active | change | commit`. I think the drag status types can be named independently from the accessible messages.

Reply:
Thought of keeping them in sync so that it's easy to understand but the suggested names are more precise and clean.